core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -466,13 +466,13 @@ private CarbonCommonConstants() {
   public static final String INDEX_COLUMNS = "INDEX_COLUMNS";
 
   /**
-   * Index handler table property. It allows user to create a new sort column from the set of
+   * Spatial index table property. It allows user to create a new sort column from the set of
    * existing schema columns. And can generate value for the new column after parsing each row
-   * through custom handler.
+   * through custom index implementation.
```

Comment:
Please add more description or rephrase. From this description, I still not know how to use it, like what should be the string to fill in for this table property, and what data type can it accept, and what custom index is (you can use `{@link className}` to link it)

Reply:
Have rephrased this description